doc/usage/repairnator-config.md

```diff
@@ -383,7 +383,7 @@ It defines the name of the docker image to use for Repairnator BEARS checkbranch
 
 Its default value is `spirals/checkbranches:latest`.
 
-### Root pathes
+### Root paths
```

Comment:
One day I'll know about that one, one day, maybe...

Reply:
hahaha